src/core-services/core/startup.js

```diff
@@ -9,51 +7,6 @@ import upsertPackages from "./util/upsertPackages.js";
  * @returns {undefined}
  */
 export default async function startup(context) {
-  const { appEvents } = context;
-
   await loadSampleData(context);
```

Comment:
Mentioned in another comment, but I'm getting errors with the sample data being loaded here. I think if we just adjust the sample data itself to set some fields differently it should be OK.